hw/dv/dpi/jtagdpi/README.md

```diff
@@ -11,3 +11,6 @@ etc.) from it.
 The `remote_bitbang` protocol is documented in the OpenOCD source tree at
 `doc/manual/jtag/drivers/remote_bitbang.txt`, or online at
 https://repo.or.cz/openocd.git/blob/HEAD:/doc/manual/jtag/drivers/remote_bitbang.txt
+In case you're wondering how to configure this interface, there is rendered 
+documentation here:
+https://openocd.org/doc/html/Debug-Adapter-Configuration.html
```

Comment:
nit: one sentence per line and [autolinks](https://opentitan.org/book/doc/contributing/style_guides/markdown_usage_style.html#links) should be avoided.

These are nits because the rest of the document isn't compliant. Feel free to ignore.

Also, the preceding sentence ending in a link doesn't have a full stop.

Reply:
Thanks, I've added a commit reformatting the file.